util/include/PerfMetrics.hpp

```diff
@@ -16,21 +16,31 @@
 #include <chrono>
 #include <variant>
 #include <mutex>
+#include <iostream>
+#include <sstream>
+#include <array>
 
 #include "RollingAvgAndVar.hpp"
 #include "Metrics.hpp"
 #include "Logger.hpp"
 
+
 using namespace std;
```

Comment:
Remove the using namespace std in header file.

Reply:
will do